src/test-helpers/typeFactory.ts

```diff
@@ -81,6 +81,34 @@ export class TypeFactory {
 		return key.setMeta(storageEntry.creator.meta);
 	}
 
+	/**
+	 * Similar to the storageKey method above, this is specific to storageKeys
+	 * that take in double map for keys.
+	 *
+	 * @param index The id to assign the key to.
```

Comment:
What do you mean by `id` here? As in what is `id` to the keys of a double map?

Reply:
I changed this up to reflect its actual meaning. You're correct that it is confusing. The `index` is just the value assigned to `indexTypeOne`.

Reply:
So I changed the name to value, and referenced what it is in the docs